scripts/run_docker_build.sh

```diff
@@ -20,16 +20,17 @@ CONDARC
 cat << EOF | docker run -i \
                         -v ${REPO_ROOT}:/staged-recipes \
                         -a stdin -a stdout -a stderr \
+                        -e HOST_USER_ID=`id -u $USER` \
                         $IMAGE_NAME \
-                        bash || exit $?
+                        bash -ex || exit $?
```

Comment:
Interestingly there is one token. However, I'm pretty sure it is not used for anything at all. It is not set on PRs. So would not show up there. Also we don't build this on `master`. So it couldn't show up there either. Finally new feedstocks are all generated on Travis CI and all of the variables are included there.